recipes/tbb/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.8
```

Comment:
Actually, I just set this to 10.7 and it worked fine. Could you please change this to 10.7?